src/components/CreateCourse/CreateCourse.tsx

```diff
@@ -45,11 +82,19 @@ export const CreateCourse: React.FC = () => {
 	const [authorName, setAuthorName] = useState('');
 	const [authorNameErrorLabelValue, setAuthorNameErrorLabelValue] =
 		useState('');
-	const [authorList, setAuthorList] = useState(mockedAuthorsList);
+	const [authorList, setAuthorList] = useState([]);
 	const [courseAuthorList, setCourseAuthorList] = useState([]);
 	const [courseAuthorListErrorLabelValue, setCourseAuthorListErrorLabelValue] =
 		useState('');
 
+	const authors = useSelector(
+		(state: RootState) => state.authorReducer.authors
+	);
+
+	useEffect(() => {
+		setAuthorList(authors);
+	}, authors);
```

Comment:
please, install this eslint plugin: https://www.npmjs.com/package/eslint-plugin-react-hooks, It will help you with react hooks), because useEffect gets an array of dependencies, not just one value